Sources/Sentry/SentryCrashReportConverter.m

```diff
@@ -147,6 +148,36 @@ - (SentryEvent *_Nullable)convertReportToEvent
     return nil;
 }
 
+/**
+ * storage_size and free_storage used `NSFileSystemFreeSize` and `NSFileSystemSize` internally,
+ * which Apple forbids sending off device; see
+ * https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api?language=objc
+ * While current crash reports no longer include this information, we want to remove it from older
+ * crash reports to respect Apple's privacy policy, which is mentioned in the link above.
+ */
+- (NSDictionary<NSString *, id> *)filteredUserContextContext
+{
+    if (self.userContext[@"context"] == nil) {
+        return nil;
+    }
+
+    NSMutableDictionary<NSString *, id> *context =
+        [[NSMutableDictionary alloc] initWithDictionary:self.userContext[@"context"]];
+
+    if (context[@"device"] == nil) {
+        return context;
+    }
+
+    NSMutableDictionary<NSString *, id> *deviceContext =
+        [[NSMutableDictionary alloc] initWithDictionary:context[@"device"]];
+    [deviceContext removeObjectForKey:@"storage_size"];
+    [deviceContext removeObjectForKey:@"free_storage"];
+
+    context[@"device"] = deviceContext;
+
+    return context;
+}
```

Comment:
l: I would not include this. The report was created by a version of the app that was not restricted by this new Apple requirements. The new binary of the app will not make use of the free space API, which could be a reason for the app to be rejected in the app store. Additionally, this part of the code will be useless after a while, and we will forget it here forever.

Reply:
Yes, that makes sense. I wanted to be extra accurate, but I think we can remove this. Removes some complexity. Thanks 👍